core/src/com/mygdx/game/core/model/GameState.java

```diff
@@ -21,6 +21,7 @@ public class GameState {
   public GameState(@NonNull MapInitializer mapInitializer,
                    @NonNull StartUnitInitializer startUnitInitializer) {
     fields = mapInitializer.initializeMap();
-    startUnitInitializer.initializeTestUnit(fields.get(Coordinates.of(0, 0)));
+    if (fields.containsKey(Coordinates.of(0, 0)))
```

Comment:
Nie jestem pewny czy if tutaj jest czymś co chcemy, bo to nam ukrywa błąd w kodzie, a bez tego ifa to dostaniemy wyjątek w gecie.